events-api/src/modules/events/events.service.ts

```diff
@@ -1,10 +1,26 @@
 import { EventModel } from "#/modules/events/events.model";
 import { EventsRepository } from "#/modules/events/events.repository";
+import { EventCreateDto } from "./dto/requests/events-create.dto";
 
 export class EventsService {
   static async getEvents(): Promise<EventModel[]> {
     const events = await EventsRepository.getAll();
 
     return events;
   }
+
+  static async createEvent(newEvent: EventCreateDto): Promise<void> {
+    await EventsRepository.create(newEvent);
```

Comment:
This method should return the created event instead of returning nothing

Reply:
done